From 917ebcd4f6fc93fdfe4d08b51c39c50ef00c2b71 Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Tue, 29 Dec 2020 06:45:02 +0000
Subject: [PATCH 17/21] ARM: dts: backport gpio binding and ethernet assert
 timing fixes for meson-gxbb-wetek.dtsi

These changes are not in the Linux 5.10 device tree files.
---
 arch/arm/dts/meson-gxbb-wetek.dtsi | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/meson-gxbb-wetek.dtsi b/arch/arm/dts/meson-gxbb-wetek.dtsi
index ad812854a1..10a4320885 100644
--- a/arch/arm/dts/meson-gxbb-wetek.dtsi
+++ b/arch/arm/dts/meson-gxbb-wetek.dtsi
@@ -6,6 +6,7 @@
  */
 
 #include "meson-gxbb.dtsi"
+#include <dt-bindings/gpio/gpio.h>
 
 / {
 	aliases {
@@ -147,7 +148,7 @@
 			reg = <0>;
 
 			reset-assert-us = <10000>;
-			reset-deassert-us = <30000>;
+			reset-deassert-us = <80000>;
 			reset-gpios = <&gpio GPIOZ_14 GPIO_ACTIVE_LOW>;
 
 			interrupt-parent = <&gpio_intc>;
-- 
2.17.1

